File: University/FactorialS1.cs
using System;
using System.IO;
public class Main
{
    public static void Main(string[] args)
    {
        var scanner =  "Inputs";
        var num = Convert.ToInt64(Console.ReadLine());
        Console.Write(Main.fact(num));
    }
    public static long fact(int num)
    {
        if (num == 0)
        {
            return 1;
        }
        else
        {
            return num * Main.fact(num - 1);
        }
    }
}

File: University/FactorialS1.php
<?php

class Main
{
    function __construct(){
    }
    public static function Main_1()
    {
        $local_this = new Main();
        return $local_this;
    }
    public static function main_1(&$args)
    {
        $scanner = "Inputs";
        $num = (int)readline();
        printf("%d",Main::fact($num));
    }
    public static function fact($num)
    {
        if ($num == 0)
        {
            return 1;
        }
        else
        {
            return $num * Main::fact($num - 1);
        }
    }
}
Main::main_1($argv);
